docs/tutorials/inference/latent_distribution_test.ipynb

```diff
@@ -9,7 +9,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
im not sure what exactly you wanted to change in this notebook. but notebooks need to be cleared before putting them into our docs (the notebooks get run automatically during the build process). by cleared I mean that none of the cells should be run, in jupyter notebook there should be a "clear all" option or something like that

Reply:
Oh, it must have changed when I was trying to find the phantom bug in this notebook.  Let me fix...